### PR TITLE
Few more replacements of brainomics in the examples

### PR DESCRIPTION
**Pull request by KamalakerDadi**

### CODE REVIEW

**Comment by KamalakerDadi** [comment]
Still these two needs to be replaced. Waiting for your ideas as it requires more samples

 examples/05_advanced/plot_localizer_mass_univariate_methods.py
 examples/05_advanced/plot_localizer_simple_analysis.py

**Comment by KamalakerDadi** [comment]
>An alternative to that one would be to send brainomics to osf and have a fetcher restricted to 1 T1map and 1 activation image from the dataset 

I can upload them. But this could cost the disk space anyway ?

How about these examples ? 

examples/05_advanced/plot_localizer_mass_univariate_methods.py
examples/05_advanced/plot_localizer_simple_analysis.py

**Comment by bthirion** [comment]
On 12/10/2018 21:47, KamalakerDadi wrote:
>
>     An alternative to that one would be to send brainomics to osf and
>     have a fetcher restricted to 1 T1map and 1 activation image from
>     the dataset
>
> I can upload them. But this could cost the disk space anyway ?
>
True, but only one subject.
>
> How about these examples ?
>
> examples/05_advanced/plot_localizer_mass_univariate_methods.py
> examples/05_advanced/plot_localizer_simple_analysis.py
>
I think that those are more in the scope of  Nistats. We may remove them 
temporarily, but at some point all this will probably get back to Nilearn.

**Comment by GaelVaroquaux** [comment]
Can you merge or rebase with master: I would love to see this run on CircleCI to see the results.

**Comment by KamalakerDadi** [comment]
> An alternative to that one would be to send brainomics to osf and
     have a fetcher restricted to 1 T1map and 1 activation image from
     the dataset

 >I can upload them. But this could cost the disk space anyway ?
>True, but only one subject.

This comment is done. Have to wait for build to see.

What we decide on this comment ?


 >How about these examples ?

 >examples/05_advanced/plot_localizer_mass_univariate_methods.py
 >examples/05_advanced/plot_localizer_simple_analysis.py
>I think that those are more in the scope of  Nistats. We may remove them
temporarily, but at some point all this will probably get back to Nilearn.

File: examples/plot_3d_and_4d_niimg.py [hunk 1]
```diff
@@ -16,13 +16,13 @@
 print('Datasets are stored in: %r' % datasets.get_data_dirs())
 
 ###############################################################################
-# Let's now retrieve a motor contrast from a localizer experiment
-tmap_filenames = datasets.fetch_localizer_button_task()['tmaps']
-print(tmap_filenames)
+# Let's now retrieve a motor contrast from a Neurovault repository
+motor_images = datasets.fetch_neurovault_motor_task()
```
Comment (KamalakerDadi) [inline examples/plot_3d_and_4d_niimg.py]: Should I change this localizer as well or let it be neurovault ?

Comment (GaelVaroquaux) [reply examples/plot_3d_and_4d_niimg.py]: It is probably a good idea to use neurovault rather than the localizer. Let us see what this looks like.

**Comment by GaelVaroquaux** [comment]
I think that the way forward here is probably to merge this change, which seems to be an improvement, and then to craft subset of brainomics which we upload to OSF for the remaining example.

**Comment by KamalakerDadi** [comment]
>and then to craft subset of brainomics which we upload to OSF for the remaining example.

This should be done in next PR or later ?

**Comment by GaelVaroquaux** [comment]
> This should be done in next PR or later ?

Later PR. This PR is already an improvement.

**Comment by bthirion** [comment]
Yes, makes sense.

**Comment by GaelVaroquaux** [comment]
Merging. This is great. Thank you very much!

**Pull request merged**